src/ert/scheduler/event.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Union
+
+
+@dataclass
+class StartedEvent:
+    iens: int
+
+
+@dataclass
+class FinishedEvent:
+    iens: int
+    returncode: int
```

Comment:
Could you maybe add a comment, which will describe some fixed `returncode` and `signal` values and what do they mean?

Reply:
I have no clue how to do this...

```py
    # The return code of the job
    returncode: int
```
seems like really garbage documentation

Reply:
I don't get this then?
```
FinishedEvent(iens=iens, returncode=254)
```

Reply:
Forward model of `iens` finished with returncode 254.

Reply:
That's the part I understand :) But why 254, since if it means that `iff returncode==254` means failed iens, it should be documented :top:

Reply:
Changed to 127 and added a comment as to why (because `sh` uses it for file not found error)